runtime/src/lib.rs

```diff
@@ -892,6 +892,16 @@ impl pallet_verifiers::common::Config for Runtime {
     type CommonWeightInfo = Runtime;
 }
 
+impl pallet_verifiers::Config<pallet_fflonk_verifier::Fflonk> for Runtime {
+    type RuntimeEvent = RuntimeEvent;
+    type OnProofVerified = Aggregate;
+    type WeightInfo =
+        pallet_fflonk_verifier::FflonkWeight<weights::pallet_fflonk_verifier::ZKVWeight<Runtime>>;
+    type Ticket = VkRegistrationHoldConsideration;
+    #[cfg(feature = "runtime-benchmarks")]
+    type Currency = Balances;
+}
+
```

Comment:
Just some thoughts. Maybe is worth of trying to refactor a bit and move all configuration that can be moved in `common::Config`. I guess for sure `Currency`, `Ticket`. For `RuntimeEvent` I'm not sure and `OnProofVerifier` maybe is possible to implement `()` that use the one in `common::Config`, an override and accumulation.

Let me know if I can spend some hours on this.

Reply:
I'll open a ticket thanks